### DISCUSSION TITLE
Replication issue

### DISCUSSION THREAD

**Discussion by svenhezel** [dragonflydb/dragonfly#725] [Q&A] [answered]
Hi everyone, 

We are running the latest dragonfly release on Google Cloud together with Google Pubsub. We are using it as a replacement of Redis where we need replication across different clusters. 

After deployment in production we have noticed that not all keys are replicated completely. So far we could not figure out the issue and had to return to Redis. We started to believe that maybe its related that not all redis command are supported. Did anyone experience something similar or would have an idea what causes this ?

Thank you!

**Comment by romange** [comment]
Hi @svenhezel , can you provide more information, please? For example, the output of `info all` command from your master instance?
Do you remember what is the type of the keys that were not replicated and what command do you use on those keys?

**Comment by bzon** [comment, marked as answer]
This is now solved.
